### ISSUE TITLE
Non-active tabs ("Events"/"Series") does not meet minimum color contrast ratio

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#532]
There is the tab headers "Events" and "Series". The non-active tab header does not meet the minimum color contrast ratio.

Currently the ratio is about 2:1 but should be at least 4.5:1.

This has a serious impact on people with low vision.
